### DISCUSSION TITLE
I have feature suggestions for the memory section in the Glow software.

### DISCUSSION THREAD

**Discussion by OpenSoftware-World** [turkaysoft/glow#21] [Ideas]
![Ekran görüntüsü 2024-06-04 171937](https://github.com/roines45/glow/assets/143949134/5d3ebeb8-c91a-485d-a176-481edbc0f27e)

As seen in the screenshot I posted, it says the memory manufacturer is unknown. I would like to see the memory manufacturer for Glow and learn the memory models in more detail. I would be very pleased if you could do this. As LinuxUsersLinuxMint, I wish you a good day. @roines45

**Comment by turkaysoft** [comment]
Hello,

The “Unknown” in the memory section actually depends on the manufacturer. Naturally, when the memory manufacturer does not write their name in the manufacturer section of the RAM, it returns blank.

Additionally, there will be some changes in the memory section with Glow v24.09.

I'm working on other new features, but I'm not including them in the general release because they still have a lot of bugs.

Best regards.

**Reply by OpenSoftware-World** [reply]
> Hello,
> 
> The “Unknown” in the memory section actually depends on the manufacturer. Naturally, when the memory manufacturer does not write their name in the manufacturer section of the RAM, it returns blank.
> 
> Additionally, there will be some changes in the memory section with Glow v24.09.
> 
> I'm working on other new features, but I'm not including them in the general release because they still have a lot of bugs.
> 
> Best regards.

Well, again, thank you for your work and interest. I look forward to the next versions. I hope you can publish them without any problems. @roines45

**Comment by mikepanchaud** [comment]
Hi,
I just tried the Windows version from a link in Computer Active magazine. I noticed in the Memory section, it does not show the manufacturing date of my DDR3 ram modules. This informaton is shown in a similar application called 'Speccy' from Piriform, as a week number and year.. I find it quite handy to know this. My RAM is 12 years old !

Also, is it correct to show the ram module serial number in Hex ? Speccy shows this in decimal.

BTW my RAM manufacturer is correctly shown as Kingston.

**Reply by turkaysoft** [reply]
Hello,

Displaying in HEX format is old technology.
Glow has the ability to display the manufacturer name directly.
